theano/misc/tests/test_pycuda_theano_simple.py

```diff
@@ -14,19 +14,19 @@
 import theano.sandbox.cuda as cuda_ndarray
 import theano.misc.pycuda_init
 
+import pycuda
+import pycuda.driver as drv
+import pycuda.gpuarray
+
 if not theano.misc.pycuda_init.pycuda_available:
```

Comment:
Travis fails if you import pycuda before checking `theano.misc.pycuda_init.pycuda_available` because it tries to execute the test without GPU.

Please revert the previous import order and add `# noqa` to the lines checking `theano.misc.pycuda_init.pycuda_available`. For example here should be:
```python
import theano.sandbox.cuda as cuda_ndarray
import theano.misc.pycuda_init

if not theano.misc.pycuda_init.pycuda_available:  # noqa
    from nose.plugins.skip import SkipTest
    raise SkipTest('Optional theano package cuda disabled')

import pycuda
import pycuda.driver as drv
import pycuda.gpuarray
```

Please do the same for the other files too. This should fix Travis error.